### ISSUE TITLE
A button to copy a "review" link for a workout offering

### ISSUE THREAD

**Issue by ayaankazerouni** [web-cat/code-workout#177]
Allow students to copy a "review" link to their submission to a workout offering, that they can send to their instructors if they'd like to see the submission. This should students avoid sending code or screenshots of code to their instructors over email.

**Comment by Samratrpal01** [comment]
Is it still open ?

**Comment by s-edwards** [comment]
Yes, this is still open. The underlying support for such routes, plus the necessary security protections, are already in place. But this feature has not been added yet. If you'd like to contribute, we'd be happy to answer questions about those internal details to make it easier.

**Comment by Samratrpal01** [comment]
Thank you for your response. I will check it tomorrow for sure.
